prime-router/metadata/HL7/catchall/hl7/datatypes/DT/Period.yml

```diff
@@ -0,0 +1,47 @@
+# $schema: ./../../../../../json_schema/fhir/hl7-to-fhir-mapping-resource-template.json
```

Comment:
This is a so-far unique case where a `Period` is created from two `DT` datatypes. `Period.start` and `Period.end` are of the `datetime` type so dates must be converted to datetime; I have reservations about doing this, but the FHIR inventory specifies this and at least in the extensions the original date is preserved.

Reply:
That particular method is in `Hl7RelatedGeneralUtils.java` in the HL7 > FHIR converter library. Looking closer at the FHIR output, it does seem like that method is happy to take only a date and format it accordingly (`20240701` became `2024-07-01`).